### PR TITLE
feat(sdk): add default schema_version to pipeline

### PR DESCRIPTION
**Pull request by ji-yaqi**
Default to 0.0.1

### CODE REVIEW

File: sdk/python/kfp/dsl/type_utils.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def get_artifact_type_schema(
     artifact_class = artifact_class_or_type_name
 
   return pipeline_spec_pb2.ArtifactTypeSchema(
-      schema_title=artifact_class.TYPE_NAME)
+      schema_title=artifact_class.TYPE_NAME, schema_version="0.0.1")
```
Comment (neuromage) [inline sdk/python/kfp/dsl/type_utils.py:94]: Maybe we can add `VERSION` field to artifacts, and let each artifact separately manage their schema versions. Then this line can be 
```py
return pipeline_spec_pb2.ArtifactTypeSchema(
      schema_title=artifact_class.TYPE_NAME, schema_version=artifact_class.VERSION)
```

WDYT?

File: api/v2alpha1/go/pipelinespec/pipeline_spec.pb.go [hunk 2]
```diff
@@ -7,6 +7,7 @@
 package pipelinespec
 
 import (
+	status "google.golang.org/genproto/googleapis/rpc/status"
```
Comment (chensun) [inline api/v2alpha1/go/pipelinespec/pipeline_spec.pb.go:10]: revert this file?

Comment (ji-yaqi) [reply api/v2alpha1/go/pipelinespec/pipeline_spec.pb.go:10]: Shall we update this file to register the new changes in pipelineSpec? Maybe we can do it in another PR.

**Comment by ji-yaqi** [comment]
v2-samples tests are blocked by PipelineSpec update. 
Opened https://github.com/kubeflow/pipelines/pull/6401 to sync PipelineSpec.

**Pull request merged**